### ISSUE TITLE
Scroll image viewer, Image Slideshow

### ISSUE THREAD

**Issue by lexcorp** [lepoco/wpfui#853]
### Is your feature request related to a problem? Please describe

I think it would be interesting to add a control that allows creating a modern menu based on the concepts: ListBox, ListView, ScrollView, Image Slide, like this:

![](https://cdn.dribbble.com/users/78637/screenshots/4934380/habitat.gif)

### Describe the solution you'd like

The solution should allow hovering the mouse pointer over an element to auto-scroll, in addition to allowing the use of transitions with visually attractive effects.

Basically a menu like this:

![](https://cdn.dribbble.com/users/78637/screenshots/4934380/habitat.gif)

### Describe alternatives you've considered

Perhaps an improved and updated version of ListBox or ListView, which allows you to create more modern menus.
